cms/djangoapps/contentstore/views/tests/test_bulksettings.py

```diff
@@ -69,13 +67,12 @@ def create_item_of_category(self, category, parent):
             metadata[setting_type] = self.generate_random_value_for(setting_type)
 
         item = ItemFactory.create(
-            parent_location = parent.location,
-            category = category,
-            metadata = metadata
+            parent_location=parent.location,
+            category=category,
+            metadata=metadata
```

Comment:
Wherever syntactically allowed (in Python), always add a trailing comma to iterables.

By appending a trailing comma to iterables, we're able to add additional
lines/items to the collection without altering previous items. This
provides a cleaner diff when viewing the version control history.